### PR TITLE
eth/tracers: fix the issue of stateobject modified

### PR DESCRIPTION
**Pull request by jsvisa**
Signed-off-by: Delweng <delweng@gmail.com>

### CODE REVIEW

**Review by s1na — APPROVED** [review]
Ah good catch! The change looks good to me. I will wait for CI before merging.

File: eth/tracers/native/prestate.go
Comment (holiman) [inline eth/tracers/native/prestate.go:82]:
```suggestion
	fromBal := new(big.Int).Set(prestate[from].Balance))
```

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
This was green, not sure why it wasn't repoorted here: https://ci.appveyor.com/project/ethereum/go-ethereum/builds/44908415

**Pull request merged**
